src/schemas/complaint-tool/schema.js

```diff
@@ -189,39 +193,38 @@ let schema = {
             type: 'object',
             required: ['address', 'name'],
             properties: {
-              address: {
+              address: { // TRANSLATE: all international address fields into street, street2, and country
                 type: 'object',
                 required: ['street', 'city', 'state', 'country', 'postalCode'],
                 properties: {
                   street: {
                     type: 'string',
                     minLength: 1,
-                    maxLength: 126 // address + address2 length must be < 255
+                    maxLength: 126 // street + street2 length must be < 255 (INTL 80 on FE)
```

Comment:
These maxLengths add up to 249, with 5 left over for spaces and commas to be inserted. Does this breakdown of limits between seem reasonable?

We previously discussed using `street`, `street2`, `city`, and `country` for international addresses (and only removing `state` and `postal code` from international addresses), but putting `state` and `postal code` onto `street2`, but leaving `city` in the `city` field, seems like it would be unclear to read, and these limits are set up with the idea that we will also move `city` into `street2` in mind.